autoload/ale/util.vim

```diff
@@ -336,18 +336,14 @@ function! ale#util#GetMatches(lines, patterns) abort
 endfunction
 
 function! s:LoadArgCount(function) abort
-    let l:Function = a:function
-
-    redir => l:output
-        silent! function Function
-    redir END
-
-    if !exists('l:output')
+    try
+        let l:output = execute('function a:function')
+    catch /E123/
         return 0
-    endif
+    endtry
 
     let l:match = matchstr(split(l:output, "\n")[0], '\v\([^)]+\)')[1:-2]
-    let l:arg_list = filter(split(l:match, ', '), 'v:val isnot# ''...''')
+    let l:arg_list = filter(split(l:match, ', '), "v:val isnot# '...'")
```

Comment:
```suggestion
    let l:arg_list = filter(split(l:match, ', '), 'v:val isnot# ''...''')
```

I'll change this back, as I prefer it how it was. Single quote doubling for escaping is common, and double quote usage should be pretty rare.